README.md

```diff
@@ -1,50 +1,73 @@
-# Product Summary
+# VTEX Product Summary
 
-Product summary is a canonical component that any VTEX store can install. 
+## Description
+The VTEX Prodcut Summary summarises the product informations such as name, price and picture.
+This is a VTEX app that is used by Dreamstore product.
+
+:loudspeaker: **Disclaimer:** Don't fork this project, use, contribute, or open issue with your feature request
 
 ## Release schedule
 | Release  | Status              | Initial Release | Maintenance LTS Start | End-of-life | Dreamstore Compatibility
 | :--:     | :---:               |  :---:          | :---:                 | :---:       | :---: 
 | [1.x]    | **Maintenance LTS** |  2018-08-21     | 2018-11-28            | March 2019  | 1.x
 | [2.x]    | **Current Release** |  2018-11-28     |                       |             | 2.x
 
-## Continuous Integrations 
+See our [LTS policy](https://github.com/vtex-apps/awesome-io#lts-policy) for more information.
 
-### Travis CI 
-[![Build Status](https://travis-ci.org/vtex-apps/product-summary.svg?branch=master)](https://travis-ci.org/vtex-apps/product-summary)
+## Table of Contents
+- [Usage](#usage)
+  - [Blocks API](#blocks-api)
+    - [Configuration](#configuration)
+  - [Styles API](#styles-api)
+- [Troubleshooting](#troubleshooting)
 
-### Running Tests
+## Usage
 
-To run the test suit, type the following in the terminal, inside the folder `react`
+This app uses our store builder with the blocks architecture. To know more about Store Builder [click here.](https://help.vtex.com/en/tutorial/understanding-storebuilder-and-stylesbuilder#structuring-and-configuring-our-store-with-object-object)
 
-```sh
-$ npm t     # this
-$ yarn test # or this one
-```
+To use this app you need to import it in your dependencies on `manifest.json` file.
```

Comment:
Just an suggestion, because it's kinda weird 'import' something in the dependencies. 'Adding' makes more sense.

To use this app you need to add it in your `dependencies` in the `manifest.json` file.